components/rating/rating.css

```diff
@@ -0,0 +1,25 @@
+@import url("../globals.css");
+.rating-container{
+  display: flex;
+  gap:-1px;
+}
+
+.rating-container>.star{
+  margin-left:-3px;
+}
+
+.rating-container > .fa-star-o {
+  color: #f2de7a;
+}
+
+.rating-container > .star-hover {
+  color: #f2de7a;
+}
+
```

Comment:
Since you are giving same color to both classes, you can separate them with a comma and write the property once only. Also, you can create a variable for the color instead of giving the hex code directly.

Reply:
I have pushed the changes. tysm for the insight

Reply:
https://github.com/SaiArunesh/responsifyUI/pull/13/commits/83c5838a7176eae3ec715a45ad21b9d6393bce9b I have made the changes